public/index.tsx

```diff
@@ -257,7 +262,7 @@ function runApp() {
     <ImageViewerApp
       cellId={args.cellid.toString()}
       baseUrl={args.baseurl}
-      appHeight="90vh"
+      appHeight="100vh"
       canvasMargin="0 120px 0 0"
```

Comment:
Any reason not to turn this into "0" and fill the screen horizontally to?

Reply:
I'll try this and if nothing obviously bad happens will push it.